youtube_dl/extractor/bellesa.py

```diff
@@ -0,0 +1,109 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+from ..utils import ExtractorError, clean_html, try_get
+from .common import InfoExtractor
+
+
+class BellesaIE(InfoExtractor):
+    _VALID_URL = r'''https?://www\.bellesa\.co/videos/(?P<id>[0-9]+)/'''
+    _TESTS = [{
+        'url': 'https://www.bellesa.co/videos/2189/my-first-time-kissing-women',
+        'md5': '543b002f4d1e5b4a0dde243f4a43fd28',
+        'info_dict': {
+            'id': '2189',
+            'ext': 'mp4',
+            'title': 'My First Time Kissing Women',
+            'thumbnail': 'https://c.bellesa.co/dkvdbifey/image/upload/v1599024046/video_upload/2189cover.jpg',
+            'description': 'Jenna opens up about her troubles navigating a long-distance relationship. Her girlfriends ask if she’d ever be open to cheating on him but she says she would never, they’ve been together since college. Lena and Carter sit on either side of her and remind her it isn’t cheating if it’s with a girl…they start kissing and rubbing her chest, going down on her together. This girl on girl scene is so sensual, and the nerves on Jenna are real — exploring your sexuality takes courage, but there’s a lot of good vibes in this threesome.',
+            'creator': 'Bellesa Films',
+            'upload_date': '20191216',
+            'timestamp': 1576539207,
+            'duration': 721,
+            'tags': ['HD Porn', 'Porn for Women', 'Orgasm', 'Bellesa Films', 'Threesome', 'FFF', 'Girl on Girl', 'Lesbians', 'Lesbian Porn', 'Nipple Licking', 'Finger', 'Cunnilingus', 'Anilingus', 'Eating Out', 'Clit Play', 'Clit Stimulation', 'Natural Breasts', 'Face Sitting', 'Spitting'],
+            'categories': ['Girl on Girl', 'Story'],
+            'age_limit': 18,
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(
+            'https://www.bellesa.co/videos/%s/' % video_id, video_id)
+
+        # videos on this page are embedded into a container called VideoCard - if there is
+        # nothing on the page referencing a VideoCard we cannot extract the information and
+        # thus need to raise an error
+        if 'VideoCard' not in webpage:
+            title = self._html_search_regex(
+                r'<title[^>]*>(?P<title>.+?)\s+\|\s+Bellesa',
+                webpage, 'title', default=None,
+                group='title', fatal=False)
+
+            raise ExtractorError('%s said: %s' % (self.IE_NAME, clean_html(title)), expected=True)
```

Comment:
Prefer:
```suggestion
            raise ExtractorError('[%s] %s: %s' % (self.IE_NAME, video_id, clean_html(title)), expected=True)
```
or define a new method
```py
    @classmethod
    def _raise_ExtractorError(cls, video_id, msg, expected=True):
        raise ExtractorError('[%s] %s: %s' % (cls.IE_NAME, video_id, msg or 'unspecified error', expected=expected)
```
and call it here and elsewhere.